src/main/java/seedu/module/logic/commands/ListCommand.java

```diff
@@ -19,6 +19,7 @@
     public CommandResult execute(Model model) {
         requireNonNull(model);
         model.updateFilteredModuleList(PREDICATE_SHOW_ALL_MODULES);
+        model.changeDisplayedList("TrackedList");
```

Comment:
Same as above, use of magic word.

Reply:
Split changeDisplayedList to displayTrackedList and displayArchivedList.